charts/istio-endpoints/README.md

```diff
@@ -77,7 +77,7 @@ the cluster, you must specify those namespaces and services in the
 + dependencies:
 +   - name: istio-endpoints
 +     repository: https://k8s-charts.nextdoor.com
-+     version: 0.4.0
```

Comment:
Is that a mistake?

Reply:
@scohen-nd No, the diff just looks confusing because this is a diff of a diff. The README.md.gotmpl updates this version number.

Reply:
https://github.com/Nextdoor/k8s-charts/blob/0aecd727c8214fd11b70c6bf85ff2c8af90d4f2c/charts/istio-endpoints/README.md.gotmpl#L79